pkg/appAPIs/appAPIs.go

```diff
@@ -129,15 +131,50 @@ func (cli_api *AppAPI) createAppAPI(createInfo string, token string) ([]byte, er
 
 // To get all the apps information.
 func CreateApp(name string, image string, username string, password string,
-	env []string, port string, token string) error {
+	env []string, envFilePath string, port string, token string) error {
 	// Endpoint to list apps.
 	url := fmt.Sprintf(constants.APPURL)
 	var createInfo string
-	if env != nil {
+	// if both are provided then we will take union of two
+	if env != nil && envFilePath != "" {
+		sliceFromEnv, sliceMap := genEnvSlice(env)
+		sliceFromEnvFile, envMap, err := GetSliceFromEnvFile(envFilePath)
+
+		if err != nil {
+			return fmt.Errorf("%s", err)
+		}
+
+		// if a same key is found in both (envFile and env from command line, we will throw an error)
+		for key, _ := range envMap {
+			_, found := sliceMap[key]
+			if found {
+				return fmt.Errorf("Duplicate environment variable: %v found. Either remove it from env file or from command line.", key)
+			}
+		}
+
+		slice := append(sliceFromEnv, ",")
+		slice = append(slice, sliceFromEnvFile...)
+		fmt.Printf("%s", slice)
```

Comment:
Why do we need to print env variables?

Reply:
I added for my reference, forgot to remove.
Will apply this change